### ISSUE TITLE
Getting 'appendChild' of null in DirHelper

### ISSUE THREAD

**Issue by dflorey** [vaadin/vaadin-element-mixin#38]
I'm trying to use vaadin components in a bundle that I'm importing into a static web page.
I'm including the script in the head so that I can use the compoenents in the body.
I'm getting the error above as the body of the document is not yet available.
Is there a way to check for undefined so that I can use the components in a plain HTML without using script tags?

```
Uncaught TypeError: Cannot read property 'appendChild' of null
    at Function.detectScrollType (vaadin-dir-helper.js:25)
    at Module../node_modules/@vaadin/vaadin-element-mixin/vaadin-dir-mixin.js (vaadin-dir-mixin.js:13)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/@vaadin/vaadin-element-mixin/vaadin-element-mixin.js (vaadin-element-mixin.js:1)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/@vaadin/vaadin-checkbox/src/vaadin-checkbox.js (vaadin-checkbox.js:1)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/@vaadin/vaadin-checkbox/src/vaadin-checkbox-group.js (vaadin-checkbox-group.js:1)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/@vaadin/vaadin-checkbox/theme/lumo/vaadin-checkbox-group.js (vaadin-checkbox-group.js:1)
```

**Comment by dflorey** [comment]
It works when loading the bundle inside the body tag, but it would be nice if it could also be loaded in the head as this is what most html designers would do.

**Comment by web-padawan** [comment]
Same as #37. 

I will revert the related change, however the recommendation is to place scripts at the end of `body`. 

Custom Elements upgrade when dedinitions are loaded, so you don't have to put scripts in `head` to use them.

**Comment by web-padawan** [comment]
Fixed by 20339024bbe84878c7be043d081c65fb567b5e20 and released as 2.4.1, please upgrade.

**Comment by dflorey** [comment]
Thanks! Great support!
